serviceorchestration-simple/src/main/java/eu/arrowhead/serviceorchestration/service/validation/utils/OrchestrationValidation.java

```diff
@@ -48,6 +48,8 @@
 import java.util.Set;
 import java.util.UUID;
 
+import static org.hibernate.validator.internal.util.Contracts.assertTrue;
```

Comment:
This is a wrong import I think.
We usually use the org.springframework.util.Assert class.